src/interfaces/raffleForm.ts

```diff
@@ -0,0 +1,14 @@
+export interface Question {
+  title: string;
+  description: string;
+  type: string;
+  template: string;
+  isRequired: boolean;
+}
+
+export interface RaffleForm {
+  id: string;
+  title: string;
+  description: string;
+  questions: Question[];
+}
```

Comment:
Should these interfaces be in separate files according to convention?

Reply:
Yeah actually, will change that :+1:

Reply:
Done